cmd/podman/diff/diff.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func Diff(cmd *cobra.Command, args []string, options entities.DiffOptions) error {
+func Diff(_ *cobra.Command, args []string, options entities.DiffOptions) error {
```

Comment:
Should we just drop this entirely?  Or is it exposed externally and it would be an API break.